### ISSUE TITLE
Figure out Qualtrics TLF Data Access

### ISSUE THREAD

**Issue by nomadj1s** [nomadj1s/TLF#1]
@jorge-ltg and @xinghuanluo need access to TLF Data.

- [x] Specify exactly which data we have access to and which data we don't have access to
       - Previous RA downloaded a different data set than Jorge
       - UMichigan has a Qualtrics account
- [ ] Damon get password from Michael Rosenbaum
- [ ] Damon follow up with UMich for access to Qualtrics
- [ ] Follow up with Trevor
       - Is May the first time that Trevor accessed Qualtrics?
       - Find out from whom did Trevor get access to Qualtrics, it was not Damon
- [ ] Damon will review documentation and emails to see if he has access to missing data
- [ ] If needed, we will contact our former project manager Michael Rosenbaum to see if he can help clear things up

**Comment by nomadj1s** [comment]
@jorge-ltg and @xinghuanluo: Can you edit the post above to clarify which data you do have access to and which you don't?

**Comment by jorge-ltg** [comment]
### 2018 Qualtrics data and access

Michael Rosenbaum has shared access to the following Qualtrics data (which I did not have access to previously) to my Wharton email account (jtello@wharton.upenn.edu). These descriptions were written by Michael over e-mail to me this morning:

- TLF - 2018 Survey Consent Reminder: Consent forms for respondents who did not mail in their consent and were asked again over email. This is the form you want. Downloading it directly to AFS can be a bit of a pain, but the UM IT materials explain how to do this in detail.
- TLF - Survey 20180611: The web survey initial implementation that was sent over email
- TLF - Survey 20180615 Secondary Emails: The web survey implementation sent to respondents' whose primary email bounced the email back. This subset used secondary emails provided by the Michigan Department of Education.
- TLF - Survey 20180615 Regenerated Emails: The web survey implementation sent to respondents' whose primary email bounced the email back. This subset used generated emails based on the pattern the school and/or district create email addresses by.
- TLF - Survey 2018 Final: This is the goo.gl short link push-to-web distribution in the mailed survey. Each link uniquely identifies a public study ID. My data documentation describes where to link this back to the MDE record IDs. 

Any remaining surveys on Qualtrics are surveys that Lucia (prior IPA RA), Mahima, and Tedi (former PM's for Brian's group) did not have owner access to. Those surveys could not be shared with Michael. In his e-mail, Michael also mentioned that Ben, Damon, Jasmina, and Trevor all have full sharing permissions to the same surveys that he has access to. 

**To summarize:** I now have access to the specific Qualtrics survey I was looking for: TLF - 2018 Survey Consent Reminder. It is the same survey as Trevor's and it seems no new responses were captured since they have the same number of observations.

### LastPass password for teacherloanforgiveness@umich.edu

Michael shared log-in information for the account above through LastPass to both me and Damon. The log-in for this account is linked [here](https://new.wolverineaccess.umich.edu/). @nomadj1s was contacted at his account j1s@uchicago.edu with instructions on how to access that information.

**Comment by nomadj1s** [comment]
@jorge-ltg: I received the login information. This seems finish and I will close this issue.
